.github/workflows/main.yaml

```diff
@@ -39,15 +39,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
         exclude:
           # Test on Windows with only the oldest and newest Python versions
-          - os: windows-latest
-            python-version: "3.9"
```

Comment:
this should stay, right (and add 13)?

Reply:
yes yes, just bump up the versions - these are the intermediate versions excluded from the windows test matrix, so the lowest version we're testing on windows is 3.9 (i.e. no longer exclude it), and yes we also don't exclude 3.13 so are also testing on it (when `3.13` is added to the supported versions by the setup-python action)